packages/schemas/tables/scopes.sql

```diff
@@ -0,0 +1,12 @@
+create table scopes (
+  id varchar(21) not null,
+  resource_id varchar(21) not null,
+  name varchar(256) not null,
+  description text,
+  created_at timestamptz not null default(now()),
+  primary key (id),
+  constraint fk_scopes_resource_id
```

Comment:
will it be better to follow the same format as indices?
```suggestion
  constraint fk__scopes__resource_id
```